contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -651,7 +623,6 @@ def dev_pay(self, bolt11, msatoshi=None, label=None, riskfactor=None,
             "maxdelay": maxdelay,
             "exemptfee": exemptfee,
             "use_shadow": use_shadow,
-            # Deprecated.
```

Comment:
nit, this isn't related to the other stuff changes, should be part of the next patch?